src/vm/CMakeLists.txt

```diff
@@ -487,9 +487,17 @@ endif()
 if(CLR_CMAKE_PLATFORM_UNIX)
     list(APPEND VM_SOURCES_WKS_ARCH
         ${ARCH_SOURCES_DIR}/unixstubs.cpp
+        unixlowmem.cpp
     )
 endif(CLR_CMAKE_PLATFORM_UNIX)
 
+if(CLR_CMAKE_PLATFORM_UNIX)
+    list(APPEND VM_SOURCES_WKS
```

Comment:
Can you please move it to the previous "if" block since it uses the same condition?